.github/workflows/ci.yml

```diff
@@ -47,13 +48,17 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-18.04, macos-latest, windows-latest]
```

Comment:
Can `windows-latest` be removed from the os list, since there is an `exclude` of the `windows-latest` below?